src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -228,14 +228,20 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
         if (platform.social || platform.institutional) {
           if (this.signInData) {
             const { email, firstName, lastName, providerId } = this.signInData
-            this._router.navigate(['/register'], {
-              queryParams: {
-                ...platform.queryParameters,
-                email,
-                firstName,
-                lastName,
-                providerId,
-              },
+            this._user.getUserSession().subscribe((userSession) => {
+              const params = platform.queryParameters
+              if (userSession.oauthSession) {
+                params['oauth'] = ''
```

Comment:
Just curious since I have a ticket related to this OAuth parameter. 

https://trello.com/c/QGZgx9ga/6981-qa-investigate-whether-oauth-can-be-removed-from-authorization-urls

Why do we need this for? I thought it was useless but I see you are adding it

Reply:
Yes, it is used to know where to redirect if it must go to `oauth/authorize` but I will check later if we can remove it since it is not very useful or maybe we can put as a hidden parameter instead